test/assets/frames/frame.html

```diff
@@ -1,3 +1,8 @@
 <link rel='stylesheet' href='./style.css'>
 <script src='./script.js' type='text/javascript'></script>
+<style>
+div {
```

Comment:
Looks like font resolution has changed in chromium, and this HTML now has different default font on travis's VM.

Reply:
Interesting. Might be worth a PSA in the release notes.

Reply:
Interesting thought.

Font stack is already highly unpredictable and volatile, so I don't think this is worth mentioning.